src/palette/elements/Button/Button.tsx

```diff
@@ -1,36 +1,30 @@
 import { useColor } from "palette/hooks"
 import React, { ReactNode, useState } from "react"
-import { GestureResponderEvent, TouchableWithoutFeedback } from "react-native"
+import { GestureResponderEvent, TouchableWithoutFeedback, ViewStyle } from "react-native"
 import Haptic, { HapticFeedbackTypes } from "react-native-haptic-feedback"
 // @ts-ignore
 import { animated, Spring } from "react-spring/renderprops-native.cjs"
 import styled from "styled-components/native"
-import { SansSize, themeProps } from "../../Theme"
+import { ColorFuncOverload, SansSize, Theme, ThemeV3 } from "../../Theme"
 import { Box, BoxProps } from "../Box"
 import { Flex } from "../Flex"
 import { Spinner } from "../Spinner"
-import { Sans } from "../Typography"
+import { Text } from "../Text"
 
 /** Different theme variations */
-export type ButtonVariant =
-  | "primaryBlack"
-  | "primaryWhite"
-  | "secondaryGray"
-  | "secondaryOutline"
-  | "secondaryOutlineWarning"
-  | "noOutline"
+export type ButtonVariant = "primaryBlack" | "primaryWhite" | "secondaryGray" | "secondaryOutline" | "text"
```

Comment:
```suggestion
export type ButtonVariant = "primaryDark" | "primaryLight" | "secondaryFill" | "secondaryOutline" | "text"
```

I see in figma these are the names they use. Let's follow the same names, for less confusion in the future?

I'll link to Jonathan too, for confirmation.

Reply:
After talking to Jonathan, it makes sense to do `fillDark`, `fillLight`, `fillGray`, `outline`, `text`.

Reply:
Good idea! I mapped all variants in `src/palette/elements/Button/index.tsx` from V2 -> V3.